src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceManagerService.java

```diff
@@ -94,7 +94,11 @@ public void deleteProject(String googleProjectId) {
       // the response will be empty if the request is successful in the delete
       request.execute();
     } catch (IOException | GeneralSecurityException e) {
-      if (e.getMessage().equals("Cannot delete an inactive project.")) {
+      if (e instanceof GoogleJsonResponseException
```

Comment:
`catch` already does class matching so it seems awkward to use `instanceof` inside of it. Although that would result in a duplicate `throw`.. up to you to decide which is worse

Reply:
I don't understand, wouldn't adding a catch before this one allow you to catch the exception subclass?